core/services/helper/main.py

```diff
@@ -77,11 +86,32 @@ def scan_ports() -> List[ServiceInfo]:
         return list(valid_services)
 
 
-if __name__ == "__main__":
+FASTAPI = FastAPI(
```

Comment:
Python convention all-uppercase just for constants. Also FastAPI is the name of the library itself, and could led to misunderstoods. Could we rename that one to something else? The convention in FastAPI is to just call it "app".

Reply:
Done.